app/models/ExportsDeclaration.scala

```diff
@@ -135,6 +135,8 @@ case class ExportsDeclaration(
   def removeSupervisingCustomsOffice: ExportsDeclaration =
     copy(locations = locations.copy(supervisingCustomsOffice = None))
 
+  def skipInlandOrBorder: Boolean = parties.declarationHoldersData.exists(_.holders.exists(_.skipInlandOrBorder))
```

Comment:
Not sure this belongs here in the model, it is about a concern of the UI flow logic. Better off in a UI 'helper' class.